POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Views/MainPage.xaml

```diff
@@ -17,73 +17,70 @@
+                <muxc:TeachingTip.IconSource>
+                    <muxc:SymbolIconSource Symbol="Help" />
+                </muxc:TeachingTip.IconSource>
+            </muxc:TeachingTip>
+
+            <ToggleSwitch x:Name="RunStartupSwitch" Header="Run at startup" VerticalAlignment="Top"></ToggleSwitch>
                 <ToggleSwitch Header="Download updates automatically" VerticalAlignment="Top"/>
 
                 <StackPanel Orientation="Horizontal">
                     <TextBlock x:Name="RunAsUserTitle" HorizontalAlignment="Left" Margin="0,24,0,12" Text="Running as user" TextWrapping="Wrap" VerticalAlignment="Top" Style="{StaticResource TitleTextBlockStyle}"/>
                     <FontIcon x:Name="RunAsUserTitleIcon"  FontFamily="Segoe MDL2 Assets" Glyph="&#xE946;" Margin="12,8,0,0" VerticalAlignment="Center" Foreground="Blue" PointerPressed="{x:Bind OpenRunAsUserTip}"/>
                 </StackPanel>
                 <muxc:TeachingTip x:Name="RunAsUserTip"
-                                    Target="{x:Bind RunAsUserTitleIcon}"
-                                    Background="Blue"
-                                    Foreground="White"
-                                    Title="Running as user"
-                                    PreferredPlacement="Right"
-                                    Subtitle="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Facere, nulla, ipsum? Molestiae quis aliquam magni harum non?"
-                                    >
-                    <StackPanel Orientation="Horizontal" Margin="0,12,0,0" HorizontalAlignment="Right">
-                        <TextBlock Text="2 of 3"  Margin="0,0,134,0" VerticalAlignment="Center"></TextBlock>
-                        <Button Background="Blue" Foreground="White" Margin="0,0,12,0" BorderBrush="White" Content="Dismiss" Click="{x:Bind CloseTeachingTips}"></Button>
-                        <Button Background="White" Foreground="Blue" Content="Next" Click="{x:Bind OpenFinalGeneralSettingsTip}"></Button>
-                    </StackPanel>
+                                        Target="{x:Bind RunAsUserTitleIcon}"
+                                        Foreground="Black"
+                                        Title="Running as user"
+                                        PreferredPlacement="Top"
+                                        Subtitle="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Facere, nulla, ipsum? Molestiae quis aliquam magni harum non?"
+                                        CloseButtonClick="{x:Bind OpenFinalGeneralSettingsTip}"
+                                        >
                 </muxc:TeachingTip>
-                <ToggleSwitch Header="Always run as administrator (Recommended)" VerticalAlignment="Top"/>
+            <ToggleSwitch Header="Always run as administrator (Recommended)" VerticalAlignment="Top"/>
```

Comment:
If you have the time, pull out strings to a file which will be used for localization

Reply:
At this point a resource string file is not going to be a priority and probably should not block.  Typically though your PR would not be passed with hard wired strings.  For a hack-a-thon, we can be more forgiving

Reply:
This is nice to know, yes probably not a priority for now - Jodom (my coach) and are trying to do prod-level reviews even if it's not prod-level

Reply:
True @markAtMicrosoft. This is not blocking :)